modules/config.py

```diff
@@ -308,6 +308,11 @@ def get_config_item_or_set_default(key, default_value, validator, disable_empty_
     default_value=-1,
     validator=lambda x: isinstance(x, int)
 )
+use_model_cache = get_config_item_or_set_default(
+    key='use_model_cache',
+    default_value=True,
+    validator=lambda x: x == False or x == True
```

Comment:
```suggestion
    validator=lambda x: isinstance(x, bool)
```

Can only be 2 states anyways, IMO more elegant way to test.
This way is already used in the code, e.g. for default_advanced_checkbox.